test/unit/css.js

```diff
@@ -203,6 +203,59 @@ test( "css() explicit and relative values", 29, function() {
 	equal( $elem.css("opacity"), "1", "'+=0.5' on opacity (params)" );
 });
 
+test( "css() non-px relative values (gh-1711)", 17, function() {
+	var cssCurrent,
+		units = {},
+		$parent = jQuery("#nothiddendiv"),
+		$child = jQuery("#nothiddendivchild", $parent),
+		add = function( prop, val, unit ) {
+			var str = ( val < 0 ? "-=" : "+=" ) + Math.abs(val) + unit;
+			$child.css( prop, str );
+			equal(
+				Math.round( parseFloat( $child.css( prop ) ) ),
+				Math.round( cssCurrent += val * units[ prop ][ unit ] ),
+				"'" + str + "' on " + prop
+			);
+		},
+		getUnits = function( prop ) {
+			units[ prop ] = {
+				"px": 1,
+				"em": parseFloat( $child.css( "fontSize", "16px" ).css( "fontSize" )),
```

Comment:
Yeah it's more logical that way, I used the `font-size` to *set* the unit `em` and not only to find it.
But finally it's the same :D